BUILD.gn

```diff
@@ -241,6 +263,29 @@ run_node("bundle") {
   ]
 }
 
+action("bundle_hash_h") {
+  script = "//tools/sha256sum.py"
+  inputs = get_target_outputs(":bundle")
+  outputs = [
+    "$target_gen_dir/bundle/hash.h",
+  ]
+  deps = [
+    ":bundle",
+  ]
+  args = [
+    "--format",
+    "__attribute__((__unused__)) static const int dummy_%s = 0;",
```

Comment:
If you just use a comment, does it not work?
```
"/* %s */"
```

Reply:
No, comments are removed by the preprocessor.